tests/Feature/QuizTest.php

```diff
@@ -114,4 +115,33 @@ public function test_addNewQuiz_malformedDescription(): void
                     });
             });
     }
+
+    public function test_getQuizByID_success(): void
+    {
+        Answer::factory()->create();
+        $response = $this->get('/api/quizzes/1');
+        $response->assertStatus(200)
+            ->assertJson(function(AssertableJson $json) {
+                $json->hasAll(['message', 'data'])
+                    ->has('data', function(AssertableJson $json) {
+                        $json->hasAll(['id', 'name', 'description', 'questions'])
+                            ->has('questions', 1, function(AssertableJson $json) {
+                                $json->hasAll(['id', 'question', 'hint', 'points', 'quiz_id', 'answers'])
+                                    ->has('answers', 1, function(AssertableJson $json) {
+                                        $json->hasAll(['id', 'answer', 'feedback', 'correct', 'question_id']);
+                        });
+                    });
+                });
+            });
```

Comment:
This indentation is still bothering me but I'm not going to block the PR over it.

For reference, you want to line the closing brackets up with the `has([` that opened them.

```suggestion
            ->assertJson(function(AssertableJson $json) {
                $json->hasAll(['message', 'data'])
                    ->has('data', function(AssertableJson $json) {
                        $json->hasAll(['id', 'name', 'description', 'questions'])
                            ->has('questions', 1, function(AssertableJson $json) {
                                $json->hasAll(['id', 'question', 'hint', 'points', 'quiz_id', 'answers'])
                                    ->has('answers', 1, function(AssertableJson $json) {
                                        $json->hasAll(['id', 'answer', 'feedback', 'correct', 'question_id']);
                                    });
                            });
                    });
            });
```